apps/website/src/pages/index.astro

```diff
@@ -27,7 +27,7 @@ const currentPage = new URL(Astro.request.url).pathname;
   <main>
```

Comment:
I am noticing a bit of content shift as the scrollbar appears/disappears.

https://user-images.githubusercontent.com/9084735/215138184-c00846fc-42fb-4ff3-884a-43a0e6d2cf73.mov

should be fixable by using [`scrollbar-gutter: stable`](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter) on the element which scrolls

Reply:
I noticed that too before this PR. I feel this is because the scroll bar width somehow changes. This change happens only when the React demos are loaded.

To test this better, I set a width of `100px` to the `::-webkit-scrollbar` and removed `background-clip` from `::-webkit-scrollbar-thumb`.

The scrollbar automatically changes sizes only when including the react demos in the dom.

<details>
<summary>With react demos</summary>

[Video19.webm](https://user-images.githubusercontent.com/45748283/215155873-089fc931-15ca-4644-896b-80e6fd744ae5.webm)

</details>

<details>
<summary>Without react demos</summary>

![image](https://user-images.githubusercontent.com/45748283/215155923-f791ff73-9b08-4555-8252-ebb2afc2ead8.png)

</details>

Is there something unique about the demos that might be causing this?

Reply:
i think this might be because itwinui-react automatically adds global styles? but i'd say dont worry about it in this PR, it's very minor and can be handled separately